src/Swarm/Game/Entity.hs

```diff
@@ -450,9 +452,7 @@ entityInventory = hashedLens _entityInventory (\e x -> e {_entityInventory = x})
 -- Inventory
 ------------------------------------------------------------
 
--- | A convenient synonym to remind us when an 'Int' is supposed to
---   represent /how many/ of something we have.
-type Count = Int
```

Comment:
We could still keep this comment but just change `Int` to `Number`.

Reply:
Ups, I should have kept that. :+1: